internal/subshell/sscommon/sscommon.go

```diff
@@ -188,7 +175,7 @@ func runDirect(env []string, name string, args ...string) error {
 	err := runCmd.Run()
 	// silence exit code errors
 	if eerr, ok := err.(*exec.ExitError); ok {
-		return silentExitCodeError{eerr}
+		return errs.WrapExitCode(eerr, eerr.ExitCode())
```

Comment:
This is changing subshell behaviour to now no longer silence the actual error. That's going to have implications on many things, not just the installer. I don't feel comfortable making this change at this stage.

Reply:
Thanks for catching this. I was under the impression ExitCode was silent, but it was not.